ts_src/psetv2/creator.ts

```diff
@@ -54,18 +52,21 @@ export class CreatorInput {
 export class CreatorOutput {
   asset: string;
   amount: number;
-  address?: string;
+  script?: Buffer;
+  blindingPublicKey?: Buffer;
   blinderIndex?: number;
 
   constructor(
     asset: string,
     amount: number,
-    address?: string,
+    script?: Buffer,
+    blindingPublicKey?: Buffer,
```

Comment:
can you rename to `blindingPubkey` just to be consistent with the field of the pset Output?

Reply:
I thought better be more explicit being a signature for end users?